### PR TITLE
fix -> invalid regular expression group while creating new app

### PR DESCRIPTION
**Pull request by udaypydi**
## Please verify the following:
- [x] Everything works on iOS/Android
- [x] `npm test` **ava** tests pass with new tests, if relevant
- [ ] `./docs/` has been updated with your changes, if relevant

### CODE REVIEW

File: src/lib/detectInstall.js [hunk 1]
```diff
@@ -33,10 +33,10 @@ function detectInstall (context) {
 
   // the plugin we're trying to install
   let plugin = parameters.second
-  
+
   // extract the package name and (optionally) version
   let packageName, packageVersion
-  const versionSepRe = /(?<!^|\/)@/
```
Comment (udaypydi) [inline src/lib/detectInstall.js]: lookbehinds need a harmony flag on node versions <= 8.9.0

Comment (jamonholmgren) [reply src/lib/detectInstall.js]: @udaypydi Thanks for the PR! Do the tests pass with `yarn ci:test` ?

Comment (udaypydi) [reply src/lib/detectInstall.js]: @jamonholmgren all the tests pass with ``` yarn ci:test ```

**Comment by jamonholmgren** [comment]
So this will actually break Ignite for `@scoped/packages` (we apparently don't have great test coverage of this). I've implemented a more robust and fully tested version in my TypeScript branch and will be pushing this live. Thanks for your help on this @udaypydi; I'll close your PR for now, but a fix is inbound.

**Pull request closed without merging**